cloudos_cli/__main__.py

```diff
@@ -2478,6 +2481,113 @@ def move_files(ctx, source_path, destination_path, apikey, cloudos_url, workspac
         click.echo(f"[ERROR] Move operation failed: {str(e)}", err=True)
         sys.exit(1)
 
+@datasets.command(name="rename")
+@click.argument("source_path", required=True)
+@click.argument("new_name", required=True)
+@click.option('-k', '--apikey', required=True, help='Your CloudOS API key.')
+@click.option('-c', '--cloudos-url', default=CLOUDOS_URL, required=True, help='The CloudOS URL.')
+@click.option('--workspace-id', required=True, help='The CloudOS workspace ID.')
+@click.option('--project-name', required=True, help='The project name.')
+@click.option('--disable-ssl-verification', is_flag=True, help='Disable SSL certificate verification.')
+@click.option('--ssl-cert', help='Path to your SSL certificate file.')
+@click.option('--profile', default=None, help='Profile to use from the config file.')
+@click.pass_context
+def renaming_item(ctx, source_path, new_name, apikey, cloudos_url,
+                    workspace_id, project_name,
+                    disable_ssl_verification, ssl_cert, profile):
+    """
+    Rename a file or folder in a CloudOS project.
+
+    SOURCE_PATH must be a full path like 'Data/folderA/old_name.txt'
+    NEW_NAME is the new name to assign.
+    """
+    if not source_path.strip("/").startswith("Data/"):
+        click.echo("[ERROR] SOURCE_PATH must start with  'Data/'.", err=True)
+        sys.exit(1)
+    click.echo("Loading configuration profile...")
+    config_manager = ConfigurationProfile()
+    required_dict = {
+        'apikey': True,
+        'workspace_id': True,
+        'workflow_name': False,
+        'project_name': True
+    }
+
+    apikey, cloudos_url, workspace_id, workflow_name, repository_platform, execution_platform, project_name = (
+        config_manager.load_profile_and_validate_data(
+            ctx,
+            INIT_PROFILE,
+            CLOUDOS_URL,
+            profile=profile,
+            required_dict=required_dict,
+            apikey=apikey,
+            cloudos_url=cloudos_url,
+            workspace_id=workspace_id,
+            workflow_name=None,
+            repository_platform=None,
+            execution_platform=None,
+            project_name=project_name
+        )
+    )
+
+    verify_ssl = ssl_selector(disable_ssl_verification, ssl_cert)
+    # Initialize Datasets clients
+    client = Datasets(
+        cloudos_url=cloudos_url,
+        apikey=apikey,
+        workspace_id=workspace_id,
+        project_name=project_name,
+        verify=verify_ssl,
+        cromwell_token=None
+    )
+
+    parts = source_path.strip("/").split("/")
+    if len(parts) == 0:
+        click.echo("[ERROR] Invalid source path.", err=True)
+        sys.exit(1)
```

Comment:
```suggestion
```

This is not possible:
- 'Data/' -> will fail before
- 'Data//' -> will also fail before
- '' -> will also fail before
- All the real possibilities are >= 2